assets/scss/partials/_container.scss

```diff
@@ -12,7 +12,7 @@
     }
 
     .col-order {
-        padding-top: 300px;
+        //padding-top: 300px;
```

Comment:
is it still needed? otherwise you should delete it.

Reply:
```suggestion
```